src/libcprover-cpp/add_dependencies.sh

```diff
@@ -19,6 +19,14 @@ LIB_LIST=$@
 # unit name collisions which affect linking against the final artefact. For more
 # details, please look at https://github.com/diffblue/cbmc/issues/7586 .
 
+# Create a temporary folder for this script to work in.
+rm -rf add_dependencies_tmp
+mkdir add_dependencies_tmp
+cd add_dependencies_tmp
+
+# The full path of the current "root" directory
```

Comment:
:pick: I found this comment confusing on first reading. Surely `pwd` Prints the Working Directory not the root directory which would be `/`? On second reading I guess this means the root of the working tree, rather than the root of the filing system, but the overloaded terminology makes this confusing.